micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/QuickStart/QuickSetup.js

```
@@ -28,9 +34,13 @@ const FaqComponent = (props) => {
             <li key={index} style={{ listStyleType: ListTag === "ul" ? "disc" : "auto", margin: "8px 0" }}>
               {isLabelLink ? (
                 action?.label ? (
-                  <a className="quickLink" href={`${window.location.host}/${window?.globalPath}/${tenantId}/${action?.link}`} target="_blank" rel="noopener noreferrer">
-                    {t(action?.label)}:
-                  </a>
+                  <a
+                  className="quickLink"
+                  href="#"
+                  onClick={(e) => handleClick(e, `${window.location.host}/${window?.globalPath}/${tenantId}/${action?.link}`)}
+                >
```

Biome

[error] Use a button element instead of an a element.

Anchor elements should only be used for default sections or page navigation
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)